# Mission Config of the sub
# ALL UNITS IN M

tasks:
 start_gate:
  prior: [10,1.25,-1.5]
  search_alg: simple
  dist_behind_gate: 1.0
  failed_repeat: true
 bangbang_dice:
  prior: [8.5,1.0,-2.3]
  search_alg: simple
  dist_behind_gate: 1.0
  failed_repeat: false
 bangbang_roulette:
  prior: [0.5,-4.5,-2.5]
  search_alg: simple
  dist_behind_gate: 1.0
  failed_repeat: false

# All tasks to be carried out by the sub in order
mission_tasks:
 - start_gate
 - bangbang_dice
 - bangbang_roulette
